modules/ROOT/pages/project-deploy-in-dev-suite.adoc

```diff
@@ -14,13 +14,13 @@ Processes are categorized by versions and you can opt to only select the latest
 == Deploy options
 
 * Select Process configuration environment (SP Only): You can choose the environment to be used for the deployment. All the parameters values, actors, connector and actors filters implementations mappings and custom scripts belong to the configuration environment.
-* Default user name: The user that will be used to login in the Portal or your applications after the deployment
+* Default user name: The user that will be used to login in the Bonita Applications after the deployment
 * Clean BDM database before deployment: Erase all existing data in the BDM and recreate the database schema from scratch.
 * Validate before deploy: Perform a validation of selected artifacts before the deployment.
 
 == Quick access after deploy
 
-Once the deployment successful, you are able to choose the application to open, either the Portal or one of the deployed living application. +
+Once the deployment successful, you are able to choose one of the deployed living applications to open. +
 Only application that can opened by the user choose in the deploy options will be displayed.
```

Comment:
I dont get what is the message of this phrase... 

is it : For display to users, you have the choice only between deployed applications. 
or : A user can only choose to open a deployed application.

Reply:
It basically means that after you deploy something you have a dropdown that lets you choose between one of the applications that are deployed on your platform.
![image](https://user-images.githubusercontent.com/22596322/134347091-57542e09-9b25-4e8a-aa5c-499d65651909.png)
![image](https://user-images.githubusercontent.com/22596322/134347174-8c0459cf-556b-4d0f-86e0-05d6d50dd3cc.png)

Reply:
In deploy options, a user can choose which application to open from the ones deployed on the platform.

Reply:
This is after the deploy is successful, so "After the deploy is successful, a user can choose which application to open from the ones deployed on the platform." ?

Reply:
Go for "After the deployment is successful, a user can choose which application to open from the ones deployed on the platform."

Reply:
never mind. that's what you added :D thanks

Reply:
Exactly my thought

Reply:
I added it